common/Logging.kt

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Moved from `platform.jvm` to `common` as it is now used by both `assemble_jvm_platform` and `deploy_npm`

It's depended on by `Shell`.